<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Edit_commands</class>
 <widget class="QDialog" name="Edit_commands">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>702</width>
    <height>555</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Edit_Command</string>
  </property>
  <property name="styleSheet">
   <string notr="true">font: 14pt &quot;Calibri&quot;;
background-color:#F6F6F6;</string>
  </property>
  <widget class="QFrame" name="frame_ok_cans">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>468</y>
     <width>685</width>
     <height>80</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QFrame#frame_ok_cans{
	background-color: #FFFFFF;
}
QPushButton {
	background-color: #E2E2E2;
    border-radius: 10px;
    border: none;
}

QPushButton:hover {
    background-color: #D9D9D9;
}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QPushButton" name="pushButton_ok">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>468</x>
      <y>24</y>
      <width>97</width>
      <height>37</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 9pt &quot;Calibri&quot;;
color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>ОК</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_cancel">
    <property name="geometry">
     <rect>
      <x>576</x>
      <y>24</y>
      <width>97</width>
      <height>37</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 9pt &quot;Calibri&quot;;
color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>Отмена</string>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="frame_command_edit">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>709</width>
     <height>433</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QFrame{
	background-color:#FFFFFF;
}

Qlabel{
	background-color: rgb(236, 236, 236);
}

QLineEdit{
	background-color: #FFFFFF;
border: solid #D9D9D9;
border-width: 1px;
border-radius: 10px;}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="label_command">
    <property name="geometry">
     <rect>
      <x>24</x>
      <y>12</y>
      <width>61</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 9pt &quot;Calibri&quot;;
color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>Команда:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_command">
    <property name="geometry">
     <rect>
      <x>24</x>
      <y>36</y>
      <width>660</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
   </widget>
   <widget class="QLabel" name="label_param_type">
    <property name="geometry">
     <rect>
      <x>24</x>
      <y>96</y>
      <width>109</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 9pt &quot;Calibri&quot;;
color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>Тип параметра:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_param_type">
    <property name="geometry">
     <rect>
      <x>24</x>
      <y>120</y>
      <width>660</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <item>
     <property name="text">
      <string>Целочисленный параметр</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Без параметра</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_response">
    <property name="geometry">
     <rect>
      <x>24</x>
      <y>180</y>
      <width>133</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 9pt &quot;Calibri&quot;;
color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>Ожидаемый ответ:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_response">
    <property name="geometry">
     <rect>
      <x>24</x>
      <y>204</y>
      <width>660</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
   </widget>
   <widget class="QLabel" name="label_error">
    <property name="geometry">
     <rect>
      <x>24</x>
      <y>264</y>
      <width>145</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 9pt &quot;Calibri&quot;;
color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>Ожидаемая ошибка:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_error">
    <property name="geometry">
     <rect>
      <x>24</x>
      <y>288</y>
      <width>660</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
   </widget>
   <widget class="QLabel" name="label_timeout">
    <property name="geometry">
     <rect>
      <x>24</x>
      <y>360</y>
      <width>265</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 9pt &quot;Calibri&quot;;
color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>Максимальное время ожидания (в мс):</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_timeout">
    <property name="geometry">
     <rect>
      <x>24</x>
      <y>384</y>
      <width>660</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>